requirements/django.txt

```diff
@@ -6,30 +6,24 @@
 #
 appdirs==1.4.3
 backports.os==0.1.1
-boto3==1.10.0             # via fs-s3fs
-botocore==1.13.0          # via boto3, s3transfer
```

Comment:
This time all the dependencies of `boto3` vanished from this file; the `--rebuild` option to the pip-compile commands in `Makefile` is likely to fix this.